nautobot/core/urls.py

```diff
@@ -29,6 +29,7 @@
     path("virtualization/", include("nautobot.virtualization.urls")),
     # API
     path("api/", include("nautobot.core.api.urls")),
+    path("api/ui/", include("nautobot.core.api.ui.urls")),
```

Comment:
Nit: Would this be better added into the URL patterns in `/core/api/urls.py`? UI APIs are a subset of APIs of APIs, which has includes already for plugins and GraphQL.